src/main/java/com/zufar/onlinestore/security/api/SecurityPrincipalProvider.java

```diff
@@ -20,4 +22,12 @@ public UserDto get() {
                 .getPrincipal();
          return userDtoConverter.toDto(userEntity);
     }
+
+    public UUID getUserId() {
```

Comment:
maybe it's better to reuse the get() method above to avoid code duplication

Reply:
yea , good point. Pls avoid code duplication

Reply:
Fixed